Document pixelFormat for ImageData (#40149)
mdn/content: files/en-us/web/api/canvasrenderingcontext2d/getimagedata/index.md

- `settings` {{optional_inline}}
  - : An object with the following properties:
    - `colorSpace`: Specifies the color space of the image data. Can be set to `"srgb"` for the [sRGB color space](https://en.wikipedia.org/wiki/SRGB) or `"display-p3"` for the [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3).

>     - `pixelFormat`: Specifies the pixel format. Can be set to `"rgba-unorm8"` for RGBA with 8 bit per component unsigned normalized format, using a {{jsxref("Uint8ClampedArray")}}, or to `"rgba-float16"` for RGBA with 16 bits per component, using a {{jsxref("Float16Array")}}. Floating-point pixel values allow representing colors in arbitrarily wide gamuts and high dynamic range (HDR).

Reviewer: ```suggestion
    - `pixelFormat`: Specifies the pixel format. Possible values:
      - `"rgba-unorm8"`, for RGBA with 8 bit per component unsigned normalized format, using a {{jsxref("Uint8ClampedArray")}}.
      - `"rgba-float16"`, for RGBA with 16 bits per component, using a {{jsxref("Float16Array")}}. Floating-point pixel values allow representing colors in arbitrarily wide gamuts and high dynamic range (HDR).
```
